src/pages/HomePage.jsx

```diff
@@ -0,0 +1,8 @@
+import FetchCars from '../components/FetchCars';
+
+const HomePage = () => (
+  <div>
+    <FetchCars />
+  </div>
+);
+export default HomePage;
```

Comment:
Please add a title for the page to match the design. As you can see on the given design, the page has a title and description. You can use the same title, however, in places where there is "Vespa" in the title, you can change it to "Car" as our app is based on cars.
![s](https://github.com/hamidazim321/book-a-car-frontend/assets/129533107/6f56a924-7f3d-4eed-a736-087c17824ec5)